assets/template-collection.css

```diff
@@ -416,6 +418,27 @@ span.active-facets__button-inner {
   }
 }
 
+@media screen and (max-width: 989px) {
+  .active-facets {
+    margin: 0 -1.2rem -1.2rem;
+  }
+
+  .active-facets__button,
+  .active-facets__button-remove {
+    margin: 0;
+    padding: 1.2rem;
+  }
+
+  span.active-facets__button-inner {
+    padding-bottom: 0.4rem;
+    padding-top: 0.2rem;
```

Comment:
on iOS it looks not properly centered vertically: 
![](https://screenshot.click/21-08-cohrc-c0p9j.png)

Reply:
Updated